sample.env

```diff
@@ -1,12 +1,15 @@
-MONGO_DB_URI=Xx
-
-PORT=4000
-
-CLUSTER=yes
-
-BASE_URL=http://localhost:4000
-FRONTEND_URL=http://localhost:3000
-TOKEN_SECRET=XX
-
-GITHUB_CLIENT_ID=XX
-GITHUB_CLIENT_SECRET=XX
+MONGO_DB_URI=XX
+
+PORT=3000
```

Comment:
Port of `base_url` and Port are not matching

Reply:
I have done it, sorry for these lot of problem  , because this the first time to contribute to open source